mirroroperator/operator.py

```diff
@@ -69,6 +70,8 @@ def watch_registry_mirrors(self):
         # optional labels to be added to daemonsets and statefulsets
         ss_ds_labels=ast.literal_eval(os.environ.get("SS_DS_LABELS")),
         ss_ds_template_labels=ast.literal_eval(os.environ.get("SS_DS_TEMPLATE_LABELS")),
+        # optional tolerations to be added to daemonsets and statefulsets
+        ss_ds_tolerations=ast.literal_eval(os.environ.get("SS_DS_TOLERATIONS")),
```

Comment:
this should probably be documented

any validation? what happens if a badly formed configuration is provided?

Reply:
There is no validation at all. Do we need it? What level of validation?

Reply:
I've added documentation.